internal/exec/stages/files/files.go

```diff
@@ -234,7 +234,15 @@ func (lst ByDirectorySegments) Swap(i, j int) {
 }
 
 func (lst ByDirectorySegments) Less(i, j int) bool {
-	return lst[i].Depth() < lst[j].Depth()
+	return depth(lst[i].Node) < depth(lst[j].Node)
+}
+
+func depth(n types.Node) uint {
```

Comment:
maybe make Depth() take a string, move it to config/util, and give it a name like PathDepth or something?

Reply:
I'm not sure that makes sense because this is the only consumer of `depth()`. It's really just a helper function for `Less()` a few lines up and nothing more.

Reply:
Oh, sure then. I thought there was some validation that used it. /shrug